Modules/Sources/WordPressUI/Views/Users/Views/UserDetailView.swift

```diff
@@ -0,0 +1,178 @@
+import SwiftUI
+
+struct UserDetailView: View {
+
+    private let userProvider: UserDataProvider
+    private let actionDispatcher: UserManagementActionDispatcher
+    let user: DisplayUser
+
+    @StateObject
+    var viewModel: UserDetailViewModel
+
+    @Environment(\.dismiss)
+    var dismissAction: DismissAction
+
+    init(user: DisplayUser, userProvider: UserDataProvider, actionDispatcher: UserManagementActionDispatcher) {
+        self.user = user
+        self.userProvider = userProvider
+        self.actionDispatcher = actionDispatcher
+        _viewModel = StateObject(wrappedValue: UserDetailViewModel(userProvider: userProvider))
+    }
+
+    var body: some View {
+        Form {
+            Section(Strings.nameSectionTitle) {
+                LabeledContent(Strings.roleFieldTitle, value: user.role)
+                LabeledContent(Strings.firstNameFieldTitle, value: user.firstName)
+                LabeledContent(Strings.lastNameFieldTitle, value: user.lastName)
+                LabeledContent(Strings.nicknameFieldTitle, value: user.handle)
+                LabeledContent(Strings.displayNameFieldTitle, value: user.displayName)
+            }
+
+            Section(Strings.contactInfoSectionTitle) {
+                LabeledContent(Strings.emailAddressFieldTitle, value: user.emailAddress)
```

Comment:
Probably not the best way to display it. I'd suggest a vertical stack.
<img width="465" alt="Screenshot 2024-11-06 at 11 49 50 AM" src="https://github.com/user-attachments/assets/b93fd4fe-3b6b-4297-a839-bbdfb9d2c2ef">